docs/code_guidelines.md

```diff
@@ -7,78 +7,78 @@ Unless specified otherwise, we should follow the guidelines outlined in
-* **Interfaces should be defined locally** in the package that is using the interface and not in the package that
+- **Interfaces should be defined locally** in the package that is using the interface and not in the package that
   defines structs which implement it. Interfaces should also be defined as minimally as possible. This way mocks can be
   generated and used independently in each package.
-* Try to cleanly **separate concerns** and do not let implementation details spill to the caller code.
-* When naming types, always keep in mind that the type will be used with the package name. We should **write code that
+- Try to cleanly **separate concerns** and do not let implementation details spill to the caller code.
+- When naming types, always keep in mind that the type will be used with the package name. We should **write code that
   does not stutter** (e.g. use `connector.Create` instead of `connector.CreateConnector`).
-* **Pointer vs value semantics** - when defining a type, we should decide what semantic will be used for that type and
+- **Pointer vs value semantics** - when defining a type, we should decide what semantic will be used for that type and
   stick with it throughout the codebase.
-* **Avoid global state**, rather pass things explicitly between structs and functions.
+- **Avoid global state**, rather pass things explicitly between structs and functions.
 
-### Packages
+## Packages
 
 We generally follow the [Style guideline for Go packages](https://rakyll.org/style-packages/). Here is a short summary
 of those guidelines:
 
-* Organize code into packages by their functional responsibility.
-* Package names should be lowercase only (don't use snake_case or camelCase).
-* Package names should be short, but should be unique and representative. Avoid overly broad package names like `common`
+- Organize code into packages by their functional responsibility.
+- Package names should be lowercase only (don't use snake_case or camelCase).
+- Package names should be short, but should be unique and representative. Avoid overly broad package names like `common`
   and `util`.
-* Use singular package names (e.g. `transform` instead of `transforms`).
-* Use `doc.go` to document a package.
+- Use singular package names (e.g. `transform` instead of `transforms`).
+- Use `doc.go` to document a package.
 
 Additionally, we encourage the usage of the `internal` package to hide complex internal implementation details of a
 package and enforce a better separation of concerns between packages.
 
-### Logging
+## Logging
 
 We want to keep our logs as minimal as possible and reserve them for actionable messages like warnings and errors when
-something does not go as expected. Info logs are fine when booting up the app, all other successful operations should be
+something does not Go as expected. Info logs are fine when booting up the app, all other successful operations should be
```

Comment:
I think this would also be lowercase `go`:

```suggestion
something does not go as expected. Info logs are fine when booting up the app, all other successful operations should be
```